### DISCUSSION TITLE
Welcome to Team8_Collection Discussions!

### DISCUSSION THREAD

**Discussion by seanpm2001** [seanpm2001/Team8_Collection#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MRBlazed90** [comment]
Omg I think I love you <3 

Thank you been looking for storm8 "like" games forever.
